at_client/lib/src/util/at_client_util.dart

```diff
@@ -133,6 +135,14 @@ class AtClientUtil {
     }
     return atKey.key!;
   }
+
+  /// Converts the String representation of encodingType to [EncodingType]
+  EncodingType atDecoderStringToEnum(String encodingType) {
```

Comment:
once we have extension on enum..this method may not be required

Reply:
Removed the above method.